src/Analyzers/Core/CodeFixes/UseObjectInitializer/AbstractUseObjectInitializerCodeFixProvider.cs

```diff
@@ -27,7 +24,7 @@ internal abstract class AbstractUseObjectInitializerCodeFixProvider<
         TMemberAccessExpressionSyntax,
         TAssignmentStatementSyntax,
         TVariableDeclaratorSyntax>
-        : SyntaxEditorBasedCodeFixProvider
+        : ForkingSyntaxEditorBasedCodeFixProvider<TObjectCreationExpressionSyntax>
```

Comment:
this moved over to the Forking editor since it was doing the forking itself.  missed this when i added that helper a few PRs back.